### ISSUE TITLE
Add comments to codebase

### ISSUE THREAD

**Issue by Aurel300** [flintrocks/flint#2]

**Comment by Aurel300** [comment]
A lot of stuff is just not commented at all. At least ensure that public functions are described briefly. Not all the names are completely self-descriptive.

**Comment by Aurel300** [comment]
Fair enough. To be fair, in all fairness, I think this would have been more fairly assigned to Daniel and Alexander. Which indeed does not belong in this fork.

**Comment by SusanEisenbach** [comment]
I wouldn’t do a pass on everything to put comments in (Daniel and Alex substantially improved the comments). However, if you are working on something and a comment would have helped, put it in.

--
Professor Susan Eisenbach
Department of Computing
Imperial College London

On 29 Oct 2018, at 15:37, Aurel <notifications@github.com<mailto:notifications@github.com>> wrote:


Fair enough. To be fair, in all fairness, I think this would have been more fairly assigned to Daniel and Alexander. Which indeed does not belong in this fork.
